utils/src/lib.rs

```diff
@@ -114,3 +162,88 @@ pub fn rustfmt(code: String) -> Result<Vec<u8>, anyhow::Error> {
 
     Ok(formatted_code)
 }
+
+#[cfg(test)]
+mod test {
+    use ark_ff::{
+        BigInteger128, BigInteger256, BigInteger320, BigInteger384, BigInteger448, BigInteger64,
+        BigInteger768, BigInteger832, UniformRand,
+    };
+    use rand::thread_rng;
+
+    use super::*;
+
+    const ITERATIONS: usize = 64;
+
+    #[test]
+    fn test_bigint_conversion() {
```

Comment:
What about edge cases, 0 to 0, MAX to MAX?
Have you ever run it with say 10k iterations?
(not saying that we should do 10k iterations in the standard test)

Reply:
Also pls add failing tests, in case I didn't miss them.

Reply:
Added